packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -0,0 +1,172 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { createAlertDefinitionSchema } from '@linode/validation';
+import { useSnackbar } from 'notistack';
+import * as React from 'react';
+import { Controller, FormProvider, useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+
+import { ActionsPanel } from 'src/components/ActionsPanel/ActionsPanel';
+import { Box } from 'src/components/Box';
+import { Breadcrumb } from 'src/components/Breadcrumb/Breadcrumb';
+import { Paper } from 'src/components/Paper';
+import { TextField } from 'src/components/TextField';
+import { Typography } from 'src/components/Typography';
+import { useCreateAlertDefinition } from 'src/queries/cloudpulse/alerts';
+
+import { generateCrumbOverrides } from '../util';
+import { CloudPulseAlertSeveritySelect } from './GeneralInformation/AlertSeveritySelect';
+
+import type {
+  CreateAlertDefinitionPayload,
+  MetricCriteria,
+  TriggerCondition,
+} from '@linode/api-v4/lib/cloudpulse/types';
+
+const triggerConditionInitialValues: TriggerCondition = {
+  criteria_condition: '',
+  evaluation_period_seconds: '',
+  polling_interval_seconds: '',
+  trigger_occurrences: 0,
+};
+const criteriaInitialValues: MetricCriteria[] = [
+  {
+    aggregation_type: '',
+    dimension_filters: [],
+    metric: '',
+    operator: '',
+    value: 0,
+  },
+];
+export const initialValues: CreateAlertDefinitionPayload = {
+  channel_ids: [],
+  engineOption: '',
+  name: '',
+  region: '',
+  resource_ids: [],
+  rule_criteria: { rules: criteriaInitialValues },
+  service_type: '',
+  severity: '',
+  triggerCondition: triggerConditionInitialValues,
+};
+
+export interface ErrorUtilsProps {
+  errors: string | string[] | undefined;
+  touched: boolean | undefined;
+}
+export const ErrorMessage = ({ errors, touched }: ErrorUtilsProps) => {
+  if (touched && errors) {
+    return <Box sx={(theme) => ({ color: theme.color.red })}>{errors}</Box>;
```

Comment:
Change the color for the error message according to the feedback provided by the Linode Team earlier